projects/wave-core/src/lib/layers/symbology/symbology-editor/symbology-editor.component.ts

```diff
@@ -23,20 +25,26 @@ export class SymbologyEditorComponent implements OnDestroy {
     @Input() layer: Layer<AbstractSymbology> = undefined;
     validLayers: Array<Layer<AbstractSymbology>> = undefined;
     private subscriptions: Array<Subscription> = [];
+    private layerChanges = new ReplaySubject<[Layer<AbstractSymbology>, AbstractSymbology]>(1);
```

Comment:
Do you need the replay functionality of this `ReplaySubject` or is it sufficient to use a normal `Subject`?
I wonder since you subscribe right in the constructor.